src/client/client.ts

```diff
@@ -331,6 +332,8 @@ export class GraphQLTransportWSClient {
           break;
 
         case MessageTypes.GQL_CONNECTION_KEEP_ALIVE:
+          this.wasKeepAliveReceived = true;
+          setTimeout(this.checkForTimeout, this.wsTimeout);
```

Comment:
I believe that wasn't needed because it's a timeout and not an interval, however as we are setting the timeout as long as we receive keep alive messages it's better do it. Thank you, done 👍